app/models/classes/format_url.rb

```diff
@@ -0,0 +1,16 @@
+class Classes::FormatUrl
+
+  def return_formatted_url(post)
```

Comment:
this seems to be have been git added in via a code review of another branch. removed.